src/Sulu/Component/Webspace/Url.php

```diff
@@ -245,8 +245,8 @@ public function setEnvironment($environment)
      */
     public function isValidLocale($language, $country)
     {
-        return (empty($this->getLanguage()) || $this->getLanguage() === $language)
-            && (empty($this->getCountry()) || $this->getCountry() === $country);
+        return ($this->getLanguage() === $language && $this->getCountry() === $country)
+            || (empty($this->getLanguage()) && empty($this->getCountry()));
```

Comment:
Hm, there is one test demanding this line... I would say it is ok for now, although it feels a bit strange for me. But the test might be there for a reason, and I can't see how it hurts.